src/frontend/src/components/warnBox.ts

```diff
@@ -0,0 +1,36 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon } from "./icons";
+
+interface warnBoxProps {
+  title: string;
+  message: string | TemplateResult;
+  componentType: string;
+  slot?: TemplateResult;
+}
+
+export const warnBox = (
+  props: warnBoxProps = {
+    title: "Warning",
+    message: html`Somewhere unsafe string that can also contain html
+      <b>Evil HTML</b>`,
```

Comment:
What exactly is this?

Reply:
It is a default dummy template, which we don't really need. Will do away with it!